samples/csharp/end-to-end-apps/Forecasting-Sales/src/eShopDashboard/Pages/_Layout.cshtml

```diff
@@ -34,11 +34,17 @@
                     <div class="collapse navbar-collapse w-100">
                         <ul class="flex-row flex-md-column navbar-nav w-100 justify-content-between" style="font-size: 10pt;">
                             <li class="nav-item d-sm-inline mt-md-3 mb-md-1">
-                                <img src="~/images/logo_eshop.png"/>
+                                <img src="~/images/logo_eshop.png" />
                             </li>
                             <li class="nav-item mt-md-5 d-none d-md-block">
                                 <b style="font-size: 12pt;">Reports</b>
                             </li>
+                            <li class="nav-item pt-md-2 pb-md-4 @(selectedMenu==SelectedMenu.Reports_TimeSeries ? "selected" : String.Empty)">
+                                <a class="nav-link px-1 py-0 text-nowrap" asp-page="/Reports/TimeSeries">
+                                    <i class="eshop-icon icon-salesbyproduct d-none d-md-inline-block" title="Time Series"></i>
+                                    <span>Time Series</span>
+                                </a>
+                            </li>
                             <li class="nav-item mt-md-2 @(selectedMenu==SelectedMenu.Reports_Product ? "selected" : String.Empty)">
```

Comment:
Minor formatting thing, but on the left sidebar under reports (where there are links to Time Series, Product, and Country, there is an extra space between Time Series and Product compared to between Product and Country. Maybe this was intentional?

Not sure what the exact fix would be (if extra space was unintentional), but maybe this change would help (would make it like the other 2 nav items).

```suggestion
                            <li class="nav-item mt-md-2 pb-md-4 @(selectedMenu==SelectedMenu.Reports_Product ? "selected" : String.Empty)">
```